models/staging/edfi_3/base/base_ef3__learning_standards.sql

```diff
@@ -0,0 +1,34 @@
+with learning_standards as (
+    {{ source_edfi3('learning_standards') }}
+),
+renamed as (
+    select
+        tenant_code,
+        api_year,
+        pull_timestamp,
+        file_row_number,
+        filename,
+        is_deleted,
+        -- ids
+        v:id::string                              as record_guid,
+        v:learningStandardId::string              as learning_standard_id,
+        v:learningStandardIdentificationCode      as v_learning_standard_identification_code,
+        -- descriptions
+        v:learningStandardItemCode::string        as learning_standard_item_code,
+        v:learningStandardCategory                as v_learning_standard_category,
```

Comment:
I believe these should be `v:learningStandardCategoryDescriptor` and `v:learningStandardScopeDescriptor`